internal/cloudapi/v2/handler.go

```diff
@@ -224,6 +224,8 @@ func imageTypeFromApiImageType(it ImageTypes, arch distro.Arch) string {
 		return "live-installer"
 	case ImageTypesMinimalRaw:
 		return "minimal-raw"
+	case ImageTypesNetinst:
+		return "netinst"
```

Comment:
Nitpick: we tend to use `-installer` suffix for all installer image types, should we use `net-installer` instead?

Reply:
Yeah, this is a good point, I opened https://github.com/osbuild/images/pull/1986 with a quick suggestion but open for other ideas about the naming of course.